CHANGELOG.md

```diff
@@ -6,454 +6,494 @@ All notable changes to this project will be documented in this file.
+- Support for Virtual Templates
+- Version bumped to 2.3.0 in preparation for Euwe release
+- New /api/automate primary collection
+  - Enhanced to return additional product information for the About modal
+  - Bulk queries now support referencing resources by attributes
+  - Added ability to delete one’s own notifications
+  - Publish Blueprint API
+  - Update Blueprint API to store bundle info in ui_properties
+  - CRUD for Service create and orchestration template
+  - Api for refreshing automate domain from git
+  - Allow compressed IDs in resource references
+
+### Service Broker: Service UI (name change from Self Service UI)
+
+- Renamed due to expanding number of use cases
+- Adding in Arbitration Rules UI
+- Service Designer: Blueprint API
+  - Arbitration Profiles
+    - Collection of pre-defined settings
+    - Work in conjunction with the Arbitration Engine
+  - Rules Engine: API
+- Added datastore for the default settings for resourceless servers
+- Create picture
+-  Generic requests OPTION method
+-  Delete dialogs API
+-  Updated the “update” API for blueprints to be more efficient
+- Cockpit integration
+-  Added About modal
+
+### SmartState
+
+- Deployed new MiqDiskCache module for use with Microsoft Azure
+  - Scan time reduced from >20 minutes to <5 minutes
+  - Microsoft Azure backed read requests reduced from >7000 requests to <1000
+- Generalized disk LRU caching module
+  - Caching module can be used by any disk module, eliminating duplication.
+  - Can be inserted “higher” in the IO path.
+  - Configurable caching parameters (memory vs performance)
+  - Will be employed to address Azure performance and throttling issues.
+  - Other disk modules converted over time.
+- Containers: Settings for proxy environment variables
+
+### User Interface
+
+- Added mandatory Subscription field to Microsoft Azure Discovery screen
+- Added Notifications Drawer and Toast Notifications List
+- Added support for vSphere Distributed Switches
+- Added support to show child/parent relations of Orchestration Stacks
+- Added Middleware Messaging entities to topology chart
+- I18n support for UI plugins
+- Arbitration Profiles management for Service Broker
+- Re-check Authentication button added to Provider list views
+- Provisioning button added to the Templates & Images list and summary screens
+- Subtype option added to Generic Catalog Items
+- About modal added to OPS UI
+- Both UIs updated to latest PatternFly and Angular PatternFly
+- Self Service UI language selections separated from Operations UI
+- Internationalization
+  - Virtual Columns
+  - Toolbars
+  - Removed string interpolation (for better localization)
+  - Changed to use gettext’s pluralization
+- Ansible Tower Jobs moved to the Configuration tab (from Clouds/Stacks)
+- Interactivity added to C3 charts on C&U screens  
+- Settings moved to top right navigation header
+- C3 Charts fully implemented - chart interaction coming soon!
```

Comment:
No need for the coming soon bit for changelog.  However it looks like it was already added (see 3 lines up)